<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentsTableForm</class>
 <widget class="QWidget" name="PaymentsTableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PrevButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер договора</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ИД сотрудника</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Человек, который будет оплачивать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Кафедра</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Направление подготовки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContractNumberTextBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerIdTextBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ParentTextBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DepartmentTextBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProfessionTextBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddRowButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveRowButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить строку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveRowButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить запись</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
